_data/changelog.yml

```diff
@@ -1,3 +1,11 @@
+- date: 2023-07-07
+  updates:
+    - "Correction: Halls of Fallacies *can* have traps on the walls"
```

Comment:
Oof! Any chance you have pics for the wall traps page?

Reply:
PR sent!